package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jest-image-snapshot",
-  "version": "2.9.0",
+  "version": "2.9.1",
```

Comment:
No need to version yourself. We will take care of that as part of the release process.

Reply:
Got it, will remove this change